.gitignore

```diff
@@ -8,6 +8,9 @@ logs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+# Filter Logs singal files
+!experimental/examples/logs
+!experimental/packages/otlp-transformer/src/logs
```

Comment:
Why are we filtering in the src directory? Could easily see this backfiring or being confusing if someone tries to make a logs transformer.

Reply:
> Why are we filtering in the src directory? Could easily see this backfiring or being confusing if someone tries to make a logs transformer.

Sorry,  this is something I forgot to remove when I was doing the sdk implementation validation, I have removed it